db/migrate/20180313113654_change_digests_to_bigint.rb

```diff
@@ -0,0 +1,33 @@
+class ChangeDigestsToBigint < ActiveRecord::Migration[5.1]
+  def up
+    # messages digest
+    remove_index :messages, :digest
+    remove_column :messages, :digest
+    add_column :messages, :digest, :bigint, :limit => 8
```

Comment:
Would it be wiser to add a new column instead? It's interesting to start thinking about this in the context of continuous deployments and deployments in clusters.